src/evox/algorithms/mo/gde3.py

```diff
@@ -0,0 +1,90 @@
+import jax
+import jax.numpy as jnp
+from jax import vmap
+from functools import partial
+
+from evox import Algorithm, jit_class, State
+from evox.operators import (
+    non_dominated_sort,
+    crowding_distance,
+    crossover,
+)
+
+
+@jit_class
+class GDE3(Algorithm):
+    """GDE3 algorithm
+
+    link: https://ieeexplore.ieee.org/document/1554717
+    """
+
+    def __init__(
+        self,
+        lb,
+        ub,
+        n_objs,
+        pop_size,
+        F=0.49,
```

Comment:
Need docstring, at least for `F` and `CR`. Otherwise it's hard to know the meaning of these parameters.
You can check out `pipelines/uni_workflow.py` for docstring example.

Reply:
Actually these two arguments do have corresponding docstring in `operators/crossover/differential_evolution.py` (which was used here in `GDE3`). Do I still need to add docstring for them in the algorithm?

Reply:
Yes, always. You can do this in a simple way, explain these two parameters in a few sentences and then pointing out that they came from `operators/crossover/differential_evolution.py`